### PR TITLE
Kontaktu Ami

### PR DESCRIPTION
**Pull request by eraulo**
Issue no: #46
Hatama kontaktu iha pykafe tamba karik ema presisa atu kontaktu ita.

Instrusaun:
- pip install -r requirements.txt
- ./manage.py migrate

### CODE REVIEW

**Comment by eraulo** [comment]
Bom dia Kolega sira!
Maun @PeteCoward nia sujestaun katak presisa hadia style ne'ebe send ba admin nia email.
Hau halo ona no ita boot sira bele hare nia imagen iha kraik ne'e:
Screenshot from 2019-07-21 10-03-13

File: pykafe/pykafe/static/css/pykafe.css [hunk 1]
```diff
@@ -349,3 +349,21 @@ li.nav-item.active a {
     border-bottom-style: ridge;
     color: var(--white-1);
 }
+
+.form-control.form-color {
+    border: 1px solid #349DB1;
+    border-radius: 2rem;
+    font-size: 1.25rem;
```
[the remark this answers is not in the document]
Comment (eraulo) [reply pykafe/pykafe/static/css/pykafe.css]: Obrigado Eng. Ony ba ita nia ne'ebe diak no sugestaun ne'ebe ajuda hau teb-tebes.

**Comment by marobo** [comment]
Maun @eraulo, iha nee ita bele troka tan oituan bele ka, ita haree `focus` kor ba formulariu sira la hanesan, ex. ih input "Name" ho input "Email" no  margin entre "Name" ho "Email" dok liu duke margin "Subject" ho "Messages", bele haree iha ihsa screenshot kraik nee:

Screenshot from 2019-07-24 14-05-47

**Comment by eraulo** [comment]
Eng. Ony ita nia hakarak focus hanesan iha imagen karik ou muda tan oitoan.
Screenshot from 2019-07-26 07-15-46

**Pull request merged**